### ISSUE TITLE
Win10x64 VBinDiff.exe: Unable to initialize windows

### ISSUE THREAD

**Issue by dontsovcmc** [madsen/vbindiff#17]
```
$ ./VBinDiff.exe ./x0 ./x
VBinDiff 3.0_beta5, Copyright 1995-2017 Christopher J. Madsen
VBinDiff comes with ABSOLUTELY NO WARRANTY; for details type `vbindiff -L'.

VBinDiff.exe: Unable to initialize windows


```

**Comment by mkoloberdin** [comment]
This might help: https://github.com/madsen/vbindiff/issues/13#issuecomment-349057370

**Comment by akien-mga** [comment]
> This might help: [#13 (comment)](https://github.com/madsen/vbindiff/issues/13#issuecomment-349057370)

Neither 3.0_beta5 nor your provided binaries seem to work on current Windows 10 x64.

**Comment by mkoloberdin** [comment]
> > This might help: [#13 (comment)](https://github.com/madsen/vbindiff/issues/13#issuecomment-349057370)
> 
> Neither 3.0_beta5 nor your provided binaries seem to work on current Windows 10 x64.

I am unable to reproduce. Works fine for me (Windows 10 64 bit, both 64 and 32-bit versions of VBinDiff). I will rebuild the binaries shortly though and let you know, since the ones at that link are 4 years old now.

**Comment by mkoloberdin** [comment]
@akien-mga Here are the newly built binaries: https://github.com/mkoloberdin/vbindiff/releases/tag/efbf1e3

**Comment by Stolz** [comment]
I have the the issue when I run the program from Cygwin, however from Windows CMD program works fine

**Comment by mkoloberdin** [comment]
@Stolz Have you tried my binaries linked above?

**Comment by Stolz** [comment]
@mkoloberdin yes I tried them, but the error persists. So far I'm only able to launch `VBinDiff.exe` from a Windows CLI (CMD) but not from unix CLI (Cygwin)
